x-pack/plugins/reporting/public/share_context_menu/register_csv_reporting.tsx

```diff
@@ -48,26 +48,20 @@ export const csvReportingProvider = ({
       ? moment.tz.guess()
       : uiSettings.get('dateFormat:tz');
 
-  const getShareMenuItems = ({
-    objectType,
-    objectId,
-    sharingData,
-    isDirty,
-    onClose,
-  }: ShareContext) => {
-    if ('search' !== objectType) {
+  const getShareMenuItems = ({ objectType, sharingData, onClose, isDirty }: ShareContext) => {
+    const { taskType, taskConfig } = sharingData;
+    if (taskType !== 'csv') {
       return [];
     }
 
     const jobParams: JobParamsCSV = {
-      browserTimezone,
+      timerange: {
+        timezone: browserTimezone,
+        // FIXME: min, max
```

Comment:
The work to do is:
- pull global time picker filter and make it part of `searchSource` 
- make sure relative time range stays relative - for automated job execution